### DISCUSSION TITLE
Failed to create schema: Connection string is required

### DISCUSSION THREAD

**Discussion by boozedog** [supabase/supabase#21880] [Questions]
I am running local supabase using the npm package and supabase init / supabase start

All supabase containers and up and healthy

I can connect to Studio, run queries, add users, etc.

However, in Table Editor, when I try to add a new schema, I get this: "Failed to create schema: Connection string is required"

Any ideas?

![Screenshot 2024-03-08 at 06 02 14](https://github.com/supabase/supabase/assets/1410808/ea32c9ff-ea17-47c0-87cf-cd779e93d27f)

**Comment by boozedog** [comment]
`CREATE SCHEMA` works in the SQL editor

**Comment by aexvir** [comment]
running supabase using the docker compose deployment and also hitting this issue

**Comment by levifry** [comment]
also having this issue from the local docker environment

**Comment by Cheizr** [comment]
Experiencing this also

![image](https://github.com/supabase/supabase/assets/22793954/4921c098-b98f-435e-956d-faa51b663065)

**Reply by activenode** [reply]
same

**Comment by activenode** [comment]
I can see two weird things which lead to that error but haven't found why exactly yet.

First, there're 2 definitions of ProjectContextProvider (which I find interesting because I'd assume there should only be one as it's only one project)

<img width="413" alt="image" src="https://github.com/supabase/supabase/assets/9058762/8818491b-496d-428f-9247-9c451efb1ea8">


But more interestingly, the connection string in that context is actually empty, hence the error message:
<img width="578" alt="image" src="https://github.com/supabase/supabase/assets/9058762/e2454ce2-508a-4da4-8ea5-b976bb09faba">

**Comment by silentworks** [comment]
There might have been a regression in the CLI. Can you all state what version you are using please? also please state it in the reply of this message where it says `Write a reply` and not create a new message.

**Reply by activenode** [reply]
@silentworks I already told @joshenlim the exact place where and why this is happening, he provided a fix just now https://github.com/supabase/supabase/pull/22229 - not a cli issue

**Reply by talsan74** [reply]
still i am getting the same in docker compose deployment

![image](https://github.com/supabase/supabase/assets/122008280/d5b73e82-7a0c-4448-b405-6740624d4f04)

**Reply by Cheizr** [reply]
I'm using latest version at 26/03/2024 cannot find the version anywhere in the self hosted version ui sry.

You can create the schemas manually but not using the ui
